### ISSUE TITLE
materialized_only must be quoted if set to false

### ISSUE THREAD

**Issue by MartinLoeper** [sdebruyn/dbt-timescaledb#12]
I am wondering whether it is expected that the `materialized_only` attribute for a continuous aggregate has to be a quoted boolean.

In other words, the following works:

```
{{
  config(
    materialized='continuous_aggregate', 
    materialized_only="False"
  )
}}
```

The following does not work, but I expected it to work:
```
{{
  config(
    materialized='continuous_aggregate', 
    materialized_only=False
  )
}}
```

I think the reason being that `config.get(...)` returns false if the attribute is set to *False*, see: https://github.com/sdebruyn/dbt-timescaledb/blob/4eeacadbfeb4482da102c3452460a40cc453c4ea/dbt/include/timescaledb/macros/relations/continuous_aggregate.sql#L13

I am not very familiar with the python and dbt ecosystem, so here is my question: Is my expectation justified or is the former version the mainstream one?

**Comment by sdebruyn** [comment]
Good catch! I clarified this in the documentation. I don't know right away how I'd fix this as the default value of a boolean is false as well.
